proxy/FetchSM.h

```diff
@@ -154,37 +140,37 @@ class FetchSM : public Continuation
   bool check_connection_close();
   int dechunk_body();
 
-  int recursion;
-  PluginVC *http_vc;
-  VIO *read_vio;
-  VIO *write_vio;
-  MIOBuffer *req_buffer;
-  IOBufferReader *req_reader;
-  char *client_response;
-  int client_bytes;
-  MIOBuffer *resp_buffer; // response to HttpConnect Call
-  IOBufferReader *resp_reader;
-  Continuation *contp;
+  int recursion               = 0;
+  PluginVC *http_vc           = nullptr;
+  VIO *read_vio               = nullptr;
+  VIO *write_vio              = nullptr;
+  MIOBuffer *req_buffer       = nullptr;
+  IOBufferReader *req_reader  = nullptr;
+  char *client_response       = nullptr;
+  int client_bytes            = -1;
+  MIOBuffer *resp_buffer      = nullptr; // response to HttpConnect Call
+  IOBufferReader *resp_reader = nullptr;
+  Continuation *contp         = nullptr;
   Ptr<ProxyMutex> cont_mutex;
   HTTPParser http_parser;
   HTTPHdr client_response_hdr;
   ChunkedHandler chunked_handler;
   TSFetchEvent callback_events;
-  TSFetchWakeUpOptions callback_options;
-  bool req_finished;
-  bool header_done;
-  bool is_method_head;
-  bool is_internal_request;
-  bool destroyed;
+  TSFetchWakeUpOptions callback_options = NO_CALLBACK;
+  bool req_finished                     = false;
+  bool header_done                      = false;
+  bool is_method_head                   = false;
+  bool is_internal_request              = true;
+  bool destroyed                        = false;
   IpEndpoint _addr;
-  int resp_is_chunked;
-  int resp_received_close;
-  int fetch_flags;
-  void *user_data;
-  bool has_sent_header;
-  int64_t req_content_length;
-  int64_t resp_content_length;
-  int64_t resp_received_body_len;
+  int resp_is_chunked            = -1;
+  int resp_received_close        = -1;
+  int fetch_flags                = 0;
+  void *user_data                = nullptr;
+  bool has_sent_header           = false;
+  int64_t req_content_length     = -1;
```

Comment:
Is this intended change? `req_content_length ` was originally initialized with `0`.